odoo/addons/base/models/ir_cron.py

```diff
@@ -480,8 +480,10 @@ def _trigger_list(self, at_list):
         if not at_list:
             return
 
-        self.ensure_one()
         now = fields.Datetime.now()
+        self.ensure_one()
+        if not self.sudo().active and all(at <= now for at in at_list):
```

Comment:
If only one `at` is bigger than now, every other ones smaller than now will be still created.

Instead of doing a early return, should you filterout the list ?

```python
if not self.sudo().active:
     at_list = [at for at in at_list if at > now]
```

Or I miss something?